src/locales/pt_BR/move-trigger.json

```
@@ -3,6 +3,10 @@
   "cutHpPowerUpMove": "{{pokemonName}} diminuiu seus PS para aumentar o poder do ataque!",
   "absorbedElectricity": "{{pokemonName}} absorveu eletricidade!",
   "switchedStatChanges": "{{pokemonName}} trocou as mudanças de atributo com o alvo!",
+  "switchedTwoStatChanges": "{{pokemonName}} switched all changes to its {{firstStat}}\nand {{secondStat}} with its target!",
+  "switchedStat": "{{pokemonName}} switched {{stat}} with its target!",
+  "sharedGuard": "{{pokemonName}} shared its guard with the target!",
+  "sharedPower": "{{pokemonName}} shared its power with the target!",
   "goingAllOutForAttack": "{{pokemonName}} está arriscando tudo nesse ataque!",
```

```suggestion
  "switchedStatChanges": "{{pokemonName}} trocou as mudanças de atributo com o alvo!",
  "goingAllOutForAttack": "{{pokemonName}} está arriscando tudo nesse ataque!",
```

No more adding "fake" translations